executor/join.go

```diff
@@ -48,19 +49,22 @@ type HashJoinExec struct {
 	outerKeys   []*expression.Column
 	innerKeys   []*expression.Column
 
-	prepared        bool
-	concurrency     uint // concurrency is number of concurrent channels and join workers.
+	prepared bool
+	// concurrency is the number of partition, build and join workers.
```

Comment:
This comment means that this variable controls the size of three types of workers?

Reply:
Yep